scripts/scil_ft_visualize_collisions.py

```diff
@@ -0,0 +1,126 @@
+#! /usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Visualize collisions found through the intersection filtering process of
+fibertubes (see ft_filter_collisions.py). The previously obtained
```

Comment:
... found through the intersection filtering process of scil_ft_filter_collisions.py

Reply:
(with the --save_colliders option)